Example/PINRemoteImage Tests/PINRemoteImage_Tests.m

```diff
@@ -511,7 +512,16 @@ - (void)testload
 
 - (void)testInvalidObject
 {
-    [self.imageManager.cache cacheObjectOnDisk:@"invalid" forKey:[self.imageManager cacheKeyForURL:[self JPEGURL] processorKey:nil]];
+    NSString * const kPINRemoteImageDiskCacheName = @"PINRemoteImageManagerCache";
+    NSString *cacheURL = [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) objectAtIndex:0];
```

Comment:
`cachePath`?  The return value of this is a local file system path and not a URL.